### DISCUSSION TITLE
VS Code extension compatible with Tye2

### DISCUSSION THREAD

**Discussion by Mrxx99** [tye-v2/tye2#43] [Show and tell]
Hello, I made the VS Code extension compatible with tye2 (names are not yet changed everywhere), you can find it here: https://github.com/Mrxx99/vscode-tye2
(Was just a very small amount of changes)

Would you be intrested in moving this into this org?
Next step for me would be to remove the telemetry from it and change from azure pipelines to github actions.

**Comment by razvangoga** [comment]
sure, i've never really used it myself but if you're interested in switching it to tye 2 i'll add it

i'll create a new repo and add you to the contributors team tomorrow

**Reply by razvangoga** [reply]
@Mrxx99 i've sent you an invite to the organization'S dev team and created a repo for the extension : https://github.com/tye-v2/tye2-vscode

**Reply by Mrxx99** [reply]
hi @razvangoga thanks, I accepted the invite but still can't push to the repository.
![grafik](https://github.com/tye-v2/tye2/assets/33566379/459f7add-b593-48c6-aa0a-0da63be7db74)

**Reply by razvangoga** [reply]
main is blocked for pushing (only merges via PRs are allowed) - can you try on a branch ?

**Reply by Mrxx99** [reply]
that's what I tried, I tried to push to a new branch called tye2

**Reply by Mrxx99** [reply]
I also can't create new branches in github UI, the New branch button is missing

**Reply by razvangoga** [reply]
try again please

**Reply by Mrxx99** [reply]
Worked, pushed to tye2, but can't make a PR to main, as this branches are unrelated, maybe we should temporarily make the tye2 branch as default branch, remove the main branch and create an new main branch from tye2?

**Reply by razvangoga** [reply]
but how is your branch unrelated to main? didn't you branch from main? 

can you try a rebase? or just simply create a new branch again from main an recopy the files there and repush

**Reply by Mrxx99** [reply]
The branch was branched of from main branch of original vscode-tye repo to keep history. I solved it to do local a merge from my remote to a new tye2 branch (branched of tye2-vscode/main) with --allow-unrelated-histories

**Reply by razvangoga** [reply]
ah.. was not aware you could do that - for the main repo i just started from scratch

**Reply by Mrxx99** [reply]
I am not sure if this is the best approach, as now e.g. VSCode gitlens all point to the PR, so I think pushing the original branch and then renamig that to main would lead to cleaner history

**Reply by Mrxx99** [reply]
that would also allow me to keep my remote as a fork of the original

**Reply by razvangoga** [reply]
don't really care that much about the original git history TBH... it's still there in the original repo if need-be

**Comment by razvangoga** [comment]
We have a repo now for the vs code extension - more can be discussed there
